### DISCUSSION TITLE
Get 'Alerts' when thread moved

### DISCUSSION THREAD

**Discussion by NorgePlayz** [NamelessMC/Nameless#2177] [Ideas]
**Please describe the feature in as much detail as possible**
So this is as same as XenForo function about thread is moved so i think NamelessMC should as well. Perfect when someone apply a staff or unban so player know staff has reply it

**Comment by tadhgboyle** [comment]
Thanks! I think this is being worked on by a third-party, but hopefully will be added soon. It is def a great feature.
